### ISSUE TITLE
Can't train my own model without this error

### ISSUE THREAD

**Issue by KatieBelli** [tsurumeso/vocal-remover#111]
I want to train my own model and arranged my dataset like this:

![01](https://user-images.githubusercontent.com/109644887/180641308-a13ddc9e-0df2-4366-a21e-be891d49c8c4.PNG)

File names instruments:

![02](https://user-images.githubusercontent.com/109644887/180641330-5812d707-0001-4722-9ce7-28c6a60f157f.PNG)

File name mixtures:

![03](https://user-images.githubusercontent.com/109644887/180641343-5ae0837f-d086-4e7d-a6f5-49ff352b2ba2.PNG)

Mixtures and instruments both contain 32 FLAC-files, so 64 files in total.

There are several errors and I don't know what to do to avoid them:

![error](https://user-images.githubusercontent.com/109644887/180641607-2732edcb-66d8-4856-9114-b1e4e2a6339c.PNG)

These files are created in **each** folder:

![folder](https://user-images.githubusercontent.com/109644887/180641640-1a3654db-f12e-40f7-b496-f44b6cc9203f.PNG)

What am I doing wrong and how can I do it the right way?

**Comment by tsurumeso** [comment]
What are the versions of PyTorch and NumPy?
You can check it with the `pip freeze` command.
I am using PyTorch 1.12.0 and Numpy 1.23.1.

**Comment by KatieBelli** [comment]
appdirs==1.4.4
audioread==2.1.9
certifi==2022.6.15
cffi==1.15.1
charset-normalizer==2.1.0
colorama==0.4.5
decorator==5.1.1
defusedxml==0.7.1
ffmpeg-python==0.2.0
future==0.18.2
gitdb==4.0.9
GitPython==3.1.27
idna==3.3
iso8601==1.0.2
joblib==1.1.0
librosa==0.8.1
llvmlite==0.38.1
lxml==4.9.1
m3u8==2.0.0
mutagen==1.45.1
numba==0.55.2
numpy==1.22.4
opencv-python==4.6.0.66
packaging==21.3
Pillow==9.2.0
pooch==1.6.0
protobuf==3.15.8
pycparser==2.21
pycryptodomex==3.15.0
pyparsing==3.0.9
requests==2.28.1
resampy==0.3.1
scikit-learn==1.1.1
scipy==1.8.1
six==1.16.0
smmap==5.0.0
SoundFile==0.10.3.post1
threadpoolctl==3.1.0
torch==1.12.0
torchaudio==0.12.0
torchvision==0.13.0
tqdm==4.64.0
typing_extensions==4.3.0
urllib3==1.26.10

**Comment by KatieBelli** [comment]
> What are the versions of PyTorch and NumPy? You can check it with the `pip freeze` command. I am using PyTorch 1.12.0 and Numpy 1.23.1.

Just wanted to add that this happens when I try to train with NumPy 1.23.1. When I use the Numpy 1.22.4 then I get the error as shown in my original post.

![error num](https://user-images.githubusercontent.com/109644887/180649648-789151da-bc10-4c58-b11c-418b305b7585.PNG)

**Comment by KatieBelli** [comment]
I'm still stuck and don't know what to do to solve this problem.

**Comment by tsurumeso** [comment]
Hmm... I can't reproduce the problem.
Please refer to #110.

**Comment by KatieBelli** [comment]
> Hmm... I can't reproduce the problem. Please refer to #110.

Thank you for replying so quickly. 

May I ask (because I am doing programming for 2 weeks now):

1. "numpy >=1.21,<1.22" -> Does that mean I need to download a numpy version which is 1.21? Or 1.22? Or do I need to put that command into CMD?
2. python >3.9, <3.10 -> Again, same question but with Python.

Looking forward to your answer.

**Comment by dviralfi** [comment]
a>b means: a greater than b, and vice versa.

 You need to install numpy version greater than 1.21 but lower than 1.22 (1.21.5 works for me). That you can install with the command 
```
py -m pip install numpy==1.21.5
```

The Python interpreter version needs to be between 3.9 and 3.10, because 3.10 won't support numpy lower than 1.22.
 You can download it at the official website of Python.

(It is recommended to use virtual environment for such things and in general, please read and learn about it. I dont want you to get errors due to 2 different versions of Python installed on your system).
Quick note for later: the venv command of making specific python versin environment is : 
```

```

Good luck in your learning journey!

**Comment by KatieBelli** [comment]
Thank you! I managed to finally use **inference.py** without any errors. Had to install a few missing modules but that's alright.

Now I'm training my model. Is it normal that **# epoch 0** is still showing? 
It's been a few minutes and I can hear that my pc is working.

![training](https://user-images.githubusercontent.com/109644887/181007459-0359fda5-8704-4c6e-8d6d-474b2b01d278.PNG)

If that's alright and the training is finished how do I make vocal remover use my model when I want to use vocal remover?

And: Can I train more than 1 model, meaning have several models in my folders ready?
